arbitrum-docs/node-running/reference/ethereum-beacon-rpc-providers.mdx

```diff
@@ -35,6 +35,8 @@ To run a validator for an L2 Arbitrum chain (i.e. Arbitrum One, Arbitrum Nova, a
 | [NodeReal](https://nodereal.io/)                                            | ✅                 |                       |
 | [Quicknode](https://www.quicknode.com/docs/ethereum)                        | ✅                 | ✅                    |
 
+_Special note that Arbitrum Nitro node software is incompatible with Erigon's Consensus Layer client, [Caplin](https://erigon.tech/), due to a formatting mismatch for a specific response type (i.e. `SecondsPerSlot` is used in Caplin instead of `SECONDS_PER_SLOT`). Therefore, if any of the providers above use Caplin, errors will occur.
```

Comment:
Not sure if you intended to use an italics formatting, but there's an extra '_' character at the beginning.

Reply:
Oooo good catch 
```suggestion
_Special note that Arbitrum Nitro node software is incompatible with Erigon's Consensus Layer client, [Caplin](https://erigon.tech/), due to a formatting mismatch for a specific response type (i.e. `SecondsPerSlot` is used in Caplin instead of `SECONDS_PER_SLOT`). Therefore, if any of the providers above use Caplin, errors will occur._
```